### PR TITLE
Fabo/1337 refactored table validators tests + my delegations

### PR DESCRIPTION
**Pull request by faboweb**
Also removed search

### CODE REVIEW

**Comment by fedekunze** [comment]
conflicts need to be resolved

File: test/unit/specs/components/staking/TableValidators.spec.js [hunk 1]
```diff
@@ -1,35 +1,117 @@
...
 
-  beforeEach(() => {
-    const instance = mount(TableValidators, {
-      doBefore: ({ store }) => {
-        store.commit(`setConnected`, true)
-        store.commit(`updateWalletBalance`, {
-          denom: `atom`,
-          amount: 1337
-        })
+  const validators = [
```
Comment (sabau) [inline test/unit/specs/components/staking/TableValidators.spec.js]: same comment as in other PRs, probably I should open an issue and centralize the discussion XD

File: app/src/renderer/components/staking/TableValidators.vue [hunk 2]
```diff
@@ -30,6 +30,7 @@ export default {
     PanelSort
   },
   props: {
+    // TODO do we really need to make this a prop? the component takes everything else from the store
```
Comment (jbibla) [inline app/src/renderer/components/staking/TableValidators.vue]: probably not, we should just use the validators getter - right?

File: test/unit/specs/components/staking/TabMyDelegations.spec.js [hunk 3]
```diff
@@ -1,181 +1,246 @@
-import setup from "../../../helpers/vuex-setup"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+import { shallowMount } from "@vue/test-utils"
 import TabMyDelegations from "renderer/components/staking/TabMyDelegations"
 
-const delegates = lcdClientMock.candidates
+const delegator_addr = `cosmos15ky9du8a2wlstz6fpx3p4mqpjyrm5ctpesxxn9`
+const validators = [
```
Comment (jbibla) [inline test/unit/specs/components/staking/TabMyDelegations.spec.js]: JSON

Comment (sabau) [reply test/unit/specs/components/staking/TabMyDelegations.spec.js]: yeah we are polluting every spec file with those blobs

File: app/src/renderer/components/staking/TableValidators.vue (hunk 2 above)
Comment (faboweb) [reply app/src/renderer/components/staking/TableValidators.vue]: actually the validators are a filtered list, see the my delegations tab

File: test/unit/specs/components/staking/TabMyDelegations.spec.js (hunk 3 above)
Comment (faboweb) [reply test/unit/specs/components/staking/TabMyDelegations.spec.js]: :/ https://github.com/cosmos/voyager/pull/2081#discussion_r260249164

Comment (faboweb) [reply test/unit/specs/components/staking/TabMyDelegations.spec.js]: done

File: app/src/renderer/vuex/store.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 import Vue from "vue"
 import Vuex from "vuex"
-import merge from "lodash/merge"
+import {merge} from "lodash"
```
Comment (jbibla) [inline app/src/renderer/vuex/store.js]: GAHHH WHEN WILL OUR LINTERS WORK

**Pull request merged**